src/shared/sam/sync.ts

```diff
@@ -417,11 +417,16 @@ export async function runSamSync(args: SyncParams) {
 
     if ((parsedVersion?.compare('1.78.0') ?? 1) < 0) {
         showOnce('sam.sync.updateMessage', async () => {
+            const message = `Your current version of SAM CLI (${parsedVersion?.version}) does not include performance improvements for "sam sync". Update to 1.78.0 or higher for faster deployments.`
+            const learnMoreUrl = vscode.Uri.parse(
+                'https://aws.amazon.com/about-aws/whats-new/2023/03/aws-toolkits-jetbrains-vs-code-sam-accelerate/'
```

Comment:
This link does not work when pasting in to my browser, maybe I need to sign in?

Reply:
Link will be dead for a few hours until relevant marketing materials are released